packages/code-studio/src/styleguide/StyleGuide.tsx

```diff
@@ -71,24 +76,34 @@ function StyleGuide(): React.ReactElement {
 
         <Flex
           {...stickyProps}
-          UNSAFE_className={HIDE_FROM_E2E_TESTS_CLASS}
+          // UNSAFE_className={HIDE_FROM_E2E_TESTS_CLASS}
           marginTop={-56}
           top={20}
           gap={10}
           alignItems="end"
         >
           {hasMultipleThemes ? <ThemePicker /> : null}
           <SamplesMenu />
+          {isTestMode && (
+            <input
+              type="text"
+              placeholder="Isolate"
+              onChange={e => setTargetSection(e.target.value)}
+              value={targetSection ?? ''}
+            />
+          )}
         </Flex>
-        <Flex
-          {...stickyProps}
-          UNSAFE_className={HIDE_FROM_E2E_TESTS_CLASS}
-          top="calc(100vh - 40px)"
-          marginTop={-32}
-          marginEnd={hasMultipleThemes ? -234 : 0}
-        >
-          <GotoTopButton />
-        </Flex>
+        {!isTestMode && (
+          <Flex
+            {...stickyProps}
+            UNSAFE_className={HIDE_FROM_E2E_TESTS_CLASS}
```

Comment:
Out of curiosity, did the HIDE_FROM_E2E_TESTS_CLASS not hide this?

Reply:
Seems like I somehow commented it out, brought it back and it looks fine.